datafusion/core/tests/sql/subqueries.rs

```diff
@@ -289,6 +289,139 @@ async fn tpch_q17_correlated() -> Result<()> {
     Ok(())
 }
 
+// Fix for issue#78 join predicates from inside of OR expr also pulled up properly.
+#[tokio::test]
+async fn tpch_q19_pull_predicates_to_innerjoin() -> Result<()> {
```

Comment:
Why add the test in the `tests/sql/subqueries.rs` file?

Reply:
Other TPCH Queries related testcases are in subqueries.rs and so I have added this test also here.

Reply:
In fact, tests in `subqueries.rs` are correlated subqueries(you can know by the file name directly), not all tpch queries. 

So you need to find a proper place for the test.

Reply:
Ok, I move the tests to different file

Reply:
Perhaps we pull all the `tpch_*` queries out into a `tcph.rs` so they are easier to track.